pkg/client/vault.go

```diff
@@ -101,6 +119,49 @@ func (api *APIClient) DeleteVault(vaultName string) error {
 	return nil
 }
 
+type CreateVaultInput struct {
+	AffiliationName string   `json:"affiliationName"`
+	VaultName       string   `json:"vaultName"`
+	Permissions     []string `json:"permissions"`
+	Secrets         []Secret `json:"secrets"`
+}
+
+type CreateVaultResponse struct {
+	CreateVault Vault `json:"createVault"`
+}
+
+func (api *APIClient) CreateVault(vault Vault) (*Vault, error) {
```

Comment:
Aj, aj.  Her har vi vel løst samme sak to ganger. Ref: https://github.com/Skatteetaten/ao/pull/198

Reply:
Commiter denne først, og håndterer kodekonflikter i påfølgende merginger.